data/hai/hai reveal 4 middle.txt

```diff
@@ -1462,34 +1474,425 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			choice
+				`	"I'd rather return to the proceedings to answer their question."`
+					goto "return for meeting question"
+				`	"It was a constructive discussion."`
+					goto library
+				`	"I witnessed the test of two wills."`
+					goto library
+				`	"The meeting was poignant."`
+					goto library
+				`	"Very aggressive and confronting."`
+					goto library
+			label "aimet is about to speak"
+			# Aimet is one of the Elders who arranged Captain Last's access
+			# and doesn't want the captain to miss the turn of the tide.
+			`		~`
+			`	"Oh my! Oh! Captain <last>, I'm sorry to interrupt, but the proceedings are entering an intense phase. Teeneep is in serious trouble, but..." She touches an unseen earbud, and her expression changes to awe. "...but I think events are about to conclude though, someone is laying all their cards on the table. Do you want to return?"`
+			choice
+				`	"Yes, lets go back now."`
+					goto return4
+				`	"No, I have more research to do here."`
+					goto library
+
+			label "no more time to read"
+			`		~`
+			`	Before you can move on to the next report, the functionary interrupts you.`
+				goto goto7
+
+			label nap
+			action
+				"Hai Reveal [A11] Sound the Alarm: time" = 7
+			`	Your eyes close and the world fades to black. You see armies of lizards eating one another near squirrels who desperately hide nuts. Giant cockroaches swarm in from nowhere, breaking up the fight. The lizards part ways, leaving behind two infinite fleets of toy boats. Elves watch as owls cower behind their nests to hide from egg-stealing squirrels. Giant dragons fly across unknown expanses. People in leather costumes chant about red wastelands, often smacking lizards who nibble on their skin. A beetle, a spider, and a centaur live peacefully at a legless round table held up by strange men with sideways eyes.`
+			`	A voice keeps whispering something to you, but you can't quite understand. It seems to be coming from behind. A loud pain... a sharp pain... a furry pain...`
+			choice
+				`	"Who are you?"`
+				`	"Get out of my head!"`
+				`	(Turn around.)`
+			`	You feel a claw poking repeatedly into your shoulder, startling you awake to the sight of an overgrown squirrel wearing a formal dress. She smiles down at you, making chittering noises that sound almost like words. After a few seconds of groggy confusion, you realize it is a Hai speaking. There was a functionary assigned to you during the proceedings, but that was so long ago... centuries... eons... the darkness begins to return...`
+			`	"Captain Last! Wake up!" Her cheerful demeanor is a grating, but effective alarm clock.`
+			label goto7
+			`	"Captain <last>, the proceedings have completed, and your time here is at an end. The Elder Council voted not to punish Teeneep, probably because she is about to trigger an election. Some Elders have referred to her as Eyrisen of the Mythos," she giggles, "and if I was superstitious, I'd agree.`
```

Comment:
```suggestion
			`	"Captain <last>, the proceedings have completed, and your time here is at an end. The Elder Council voted not to punish Teeneep, probably because she is about to trigger an election. Some Elders have referred to her as Eyrisen of the Mythos," she giggles, "and if I was superstitious, I'd agree."`
```